virtcontainers/device/drivers/vfio.go

```diff
@@ -97,17 +114,38 @@ func (device *VFIODevice) DeviceType() config.DeviceType {
 	return device.DevType
 }
 
-// getBDF returns the BDF of pci device
-// Expected input strng format is [<domain>]:[<bus>][<slot>].[<func>] eg. 0000:02:10.0
-func getBDF(deviceSysStr string) (string, error) {
-	tokens := strings.Split(deviceSysStr, ":")
+type VFIODevice_Type int32
+
+const (
+        // 0 is reserved for errors.
+        VFIODevice_TYPE_VFIO_NORMAL  VFIODevice_Type = 1
+        VFIODevice_TYPE_VFIO_MDEV    VFIODevice_Type = 2
```

Comment:
please don't use snake case

Reply:
No, because of Golang rules, please use camel case only.